### ISSUE TITLE
Unable to create an OAuth connection. Reason: "Path is invalid:. Details: PowerBIRESTAPI

### ISSUE THREAD

**Issue by TheCatGeek** [migueesc123/PowerBIRESTAPI#53]
Running into this error when attempting to set up refreshing using an on-premesis gateway. 
![error](https://user-images.githubusercontent.com/48811819/113296284-41803300-92c7-11eb-99e3-ddfcc533bc0e.JPG)

**Comment by migueesc123** [comment]
Hey!

I haven’t actually seen this screen before at the dataset level or even the gateway admin level.

Could you please share some repro steps? Where exactly are you seeing this error and what you do exactly to get it.

**Comment by klinejordan** [comment]
This is a dataflow connection screen and custom connectors are not yet supported in dataflows

**Comment by TheCatGeek** [comment]
Thank you, yes, it was a dataflow. Custom connectors are in beta, but it seems they may not work fully. I was having an error with the normal dataset refresh because I did not enable load on all the tables I was using the custom connector with. After I removed those tables, refresh was able to function.

**Comment by migueesc123** [comment]
> Thank you, yes, it was a dataflow. Custom connectors are in beta, but it seems they may not work fully. I was having an error with the normal dataset refresh because I did not enable load on all the tables I was using the custom connector with. After I removed those tables, refresh was able to function.

Would you mind sharing a screenshot of the errors that you were receiving at the dataset level, please ? 
The connector should behave like any other connector in Power BI. That means that there shouldn’t be any errors specific to the connector.

**Comment by TheCatGeek** [comment]
I would, but I already fixed it. I just tried to recreate the error and couldn't do it. I've seen before where Power BI gets weird about datasources that are included in refresh but are not loaded, so it might have been a power bi bug and not the connector. Everything is working now (even when I don't enable load). If I run across this problem again I'll send a screenshot for sure.

Also THANK YOU for making this connector!!! :)
